src/webview/js/js.js

```diff
@@ -604,6 +604,10 @@ const eventUpdateHandlers = {
 };
 
 // See `handleInitialLoad` for how this gets subscribed to events.
+
+// Flow 0.98.0 (in an upcoming commit) wants this to be
```

Comment:
nit: I'd use a line like `//` to keep the above comment visually attached to the code it's commenting on, while still getting this blank line between paragraphs.

Doesn't much matter here, because this comes and goes within the span of the series.

Reply:
Sure, good idea!